src/V3Error.h

```diff
@@ -530,8 +532,12 @@ class V3Error final {
 };
 
 // Global versions, so that if the class doesn't define an operator, we get the functions anyway.
-void v3errorEnd(std::ostringstream& sstr) VL_RELEASE(V3Error::s().m_mutex);
-void v3errorEndFatal(std::ostringstream& sstr) VL_RELEASE(V3Error::s().m_mutex);
+void v3errorEnd(std::ostringstream& sstr) VL_RELEASE(V3Error::s().m_mutex) VL_MT_SAFE;
+void v3errorEndFatal(std::ostringstream& sstr) VL_RELEASE(V3Error::s().m_mutex) VL_MT_SAFE;
+
+#ifndef VL_MT_DISABLED_CODE_UNIT
+#define VL_MT_DISABLED_CODE_UNIT 0
+#endif
```

Comment:
But up above you are selecting based on #ifdef not #if, so I'm confused what is expected usage.

Reply:
With recent change (https://github.com/verilator/verilator/pull/4446) that made V3Error mutex locking function to be shared between different code units, I need to pass value of this define as function parameter, but you are right, this is confusing now.

I've changed this to always define new define based on presence of ``VL_MT_DISABLED_CODE_UNIT`` define.